pkgs/messages_builder/lib/generation_options.dart

```
@@ -65,6 +69,11 @@ class GenerationOptions {
           IndexType.integer,
       header: messagesOptions?['header'] as String? ??
           'Generated by package:messages_builder.',
+      pluralSelector: PluralSelectorType.values
+              .where((type) =>
+                  type.name == messagesOptions?['pluralSelector'] as String?)
+              .firstOrNull ??
+          PluralSelectorType.intl,
```

This will silently ignore the situation where a user has a `pluralSelector` entry but it's not one of the supported options. Is that desired? Or should this let the user know?